### DISCUSSION TITLE
Can't import the named export 'unstable_batchedUpdates' from non EcmaScript module

### DISCUSSION THREAD

**Discussion by 0xdre** [wevm/wagmi#951] [Q&A]
Hi,

I just installed wagmi and followed the overview on the home page for connecting wallet. When trying to start my react app I got a compile error with the following message:

./node_modules/wagmi/node_modules/@tanstack/react-query/build/lib/reactBatchedUpdates.mjs
Can't import the named export 'unstable_batchedUpdates' from non EcmaScript module (only default export is available)

Any ideas on how I can solve this? Thanks!

**Comment by jxom** [comment]
Hi! Can you share what kind of environment you're working in (Create React App, Next, Remix, any other framework/bundler, etc)?

**Reply by 0xdre** [reply]
Create React App

**Comment by Markeljan** [comment]
Same error with next13 latest builds.

Failed to compile.

./node_modules/@tanstack/react-query/build/lib/reactBatchedUpdates.mjs
Attempted import error: 'unstable_batchedUpdates' is not exported from 'react-dom' (imported as 'ReactDOM').

Import trace for requested module:
./node_modules/@tanstack/react-query/build/lib/reactBatchedUpdates.mjs
./node_modules/@tanstack/react-query/build/lib/setBatchUpdatesFn.mjs
./node_modules/@tanstack/react-query/build/lib/index.mjs
./node_modules/wagmi/dist/index.js
./components/chat.tsx

**Reply by alberpeb** [reply]
Im having the exact same issue right now. 
I was using react-query with next 13.1 and everything was fine. Now this came out. 

EDIT: this issue started happening to me when i upgraded to version 13.4.1

**Reply by Markeljan** [reply]
My solution was patching @tanstack/react-query like so:
https://github.com/Markeljan/dappchat/blob/main/patches/%40tanstack%2Breact-query%2B4.29.15.patch

**Reply by alberpeb** [reply]
awesome! I will be trying it. thanks!
